holonote/tests/test_annotators_element.py

```diff
@@ -137,6 +133,64 @@ def test_set_regions_multiple(multiple_annotators):
     assert output2 == expected2
 
 
+def test_single_shared_axis_hspan(annotator_range2d):
+    annotator = annotator_range2d
+    bounds = (-1, -1, 1, 1)
+    data = np.array([[0, 1], [1, 0]])
+    img = hv.Image(data, kdims=["x", "y"], bounds=bounds)
+    img_right = hv.Image(data, kdims=["z", "y"], bounds=bounds)  # y as second kdim! so HSpan
+
+    left_plot = annotator * img
+    right_plot = annotator * img_right
+    layout = left_plot + right_plot
+    hv.render(layout)
+
+    annotator.set_regions(x=(-0.15, 0.15), y=(-0.25, 0.25))
+    annotator.add_annotation(description="Test")
+
+    left_display_data = get_display_data_from_plot(left_plot, hv.Rectangles, ["x", "y"])
+    right_display_data = get_display_data_from_plot(right_plot, hv.HSpans, ["y"])
+
+    expected_left = [-0.15, -0.25, 0.15, 0.25]
+    assert (
+        left_display_data == expected_left
+    ), f"Expected {expected_left}, but got {left_display_data}"
+
+    expected_right = [-0.25, 0.25]
+    assert (
+        right_display_data == expected_right
+    ), f"Expected {expected_right}, but got {right_display_data}"
+
+
+def test_single_shared_axis_vspan(annotator_range2d):
+    annotator = annotator_range2d
+    bounds = (-1, -1, 1, 1)
+    data = np.array([[0, 1], [1, 0]])
+    img = hv.Image(data, kdims=["x", "y"], bounds=bounds)
+    img_right = hv.Image(data, kdims=["y", "z"], bounds=bounds)  # y as first kdim! so VSpan
+
+    left_plot = annotator * img
+    right_plot = annotator * img_right
+    layout = left_plot + right_plot
+    hv.render(layout)
+
+    annotator.set_regions(x=(-0.15, 0.15), y=(-0.25, 0.25))
+    annotator.add_annotation(description="Test")
+
+    left_display_data = get_display_data_from_plot(left_plot, hv.Rectangles, ["x", "y"])
+    right_display_data = get_display_data_from_plot(right_plot, hv.VSpans, ["y"])
+
+    expected_left = [-0.15, -0.25, 0.15, 0.25]
+    assert (
+        left_display_data == expected_left
+    ), f"Expected {expected_left}, but got {left_display_data}"
+
+    expected_right = [-0.25, 0.25]
+    assert (
+        right_display_data == expected_right
+    ), f"Expected {expected_right}, but got {right_display_data}"
```

Comment:
Adding a comment is not needed. 

The code itself should be explanatory.

Reply:
I think it's hard to detect the difference between these tests unless someone knows what to look for in the code, and then to interpret why the kdim position relates to the resulting element type. But I can remove it if you really think it's not needed

Reply:
nvm, I'll just remove